src/Data/Yaml/Include.hs

```diff
@@ -77,4 +81,12 @@ decodeFileEither
     :: FromJSON a
     => FilePath
     -> IO (Either ParseException a)
-decodeFileEither = decodeHelper_ . eventsFromFile
+decodeFileEither = fmap (fmap snd) . decodeFileWithWarnings
+
+-- |
```

Comment:
Would you mind adding a Haddock string?

Reply:
Done in https://github.com/snoyberg/yaml/pull/146/commits/3d29d0bf87f25936c80b29ad4cefe0afc4f7c326.